### PR TITLE
Change font

### LINKED ISSUE #11: Change font

### PR DESCRIPTION
**Pull request by toni-santos**
Closes #11 

Change the website font to a something more modern and pleasing.

image

### CODE REVIEW

**Comment by toni-santos** [comment]
Regarding this issue, this seems like all that we need to do. There may, as always, be bugs.

It would also be interesting to allow users to choose their own font, either from a selection made by us, available in the pop-up, or by inputting a font name, that we could then import from [Google Fonts](https://fonts.google.com), per example. Both of these features warrant some design decisions within the popup, and given the ongoing #98 PR this might not be the best of times to focus on this.

**Comment by thePeras** [comment]
I think there should be an option to enable/disable this feature in the popup

**Comment by toni-santos** [comment]
> I think there should be an option to enable/disable this feature in the popup.
> 
> However, I like the font a loot, it feels more comfort to read.

Just added this option and fixed a small issue on a function in the same file. Please check it out :)

**Review by thePeras — COMMENTED** [review]
Make the popup text more user friendly like "Mudar fonte de letra".

Also, now comparing to the default font, the new one seams to big. There is any possible to decrease the font-size?

File: popup/components/sections/LayoutContent.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ const LayoutContent = () => (
                 <SwitchControl
                     label="Relembrar utilizador"
                     storageKey="autoLogin"/>
+                <SwitchControl
+                    label="Mudar font"
```
Comment (ttoino) [inline popup/components/sections/LayoutContent.js:17]: I agree with João, we should have a friendlier label

**Review by ttoino — CHANGES_REQUESTED** [review]
Looks good, just a little comment

**Comment by ttoino** [comment]
> Also, now comparing to the default font, the new one seams to big. There is any possible to decrease the font-size?

IMO it looks fine like this, but we could change the font size on the `html` element, sigarra uses `em` for its font sizes thankfully.

**Review by ttoino — APPROVED** [review]
Just update the branch and this should be good to merge!

**Pull request merged**